### ISSUE TITLE
Question on scaling to large dataset

### ISSUE THREAD

**Issue by goleng** [StatMixedML/XGBoostLSS#13]
Hi Alexander, thanks for this incredible work. I really like this package; it's really helpful in situations where providing point forecast is not enough. 

However, I'm curious about scaling to large dataset. Please pardon my ignorance. I've seen the examples in the notebook section but the datasets used isn't much. 

So, my question is, does the current implementation works in similar fashion as the xgboost package? That is, one can leverage spark, dask, etc out of the box with xgboostlss? 

I'll appreciate it if you can shed some light on this.

Thanks.

**Comment by StatMixedML** [comment]
@goleng Thanks for your interest.

Concerning your questions:

> So, my question is, does the current implementation works in similar fashion as the xgboost package? That is, one can leverage spark, dask, etc out of the box with xgboostlss?

XGBoostLSS is a wrapper around the xgboost implementations. That is why XGBoostLSS comes with the same functionality as xgboost. The only requirements are that dask etc. support using custom evaluation and loss functions. If that is true you can use any of your suggestions.

**Comment by goleng** [comment]
@StatMixedML Thank you for the clarification, much appreciated.

**Comment by StatMixedML** [comment]
@edgBR Yes correct, it is a wrapper around XGBoost, so that you have access to all its functionalities. The only requirements are that _gpu_hist_ supports using custom evaluation and loss functions. If that is true you can use it.
